.gitignore

```diff
@@ -1,6 +1,6 @@
 node_modules
 dist
-lib
+# lib
```

Comment:
This is temporary while we test this repo in terra-core/terra-framework from the GitHub reference.